### ISSUE TITLE
Undefined method `parent_instance'

### ISSUE THREAD

**Issue by murdoch** [DisruptiveAngels/administrate-field-refile#2]
I'm seeing the following error when I visit my posts index page `/admin/posts` after uploading a file with your gem.

> undefined method `parent_instance' for #Administrate::Field::Refile:0x007f6ec2b56908

The error makes reference to the following:

> gems/administrate-field-refile-0.0.4/app/views/fields/refile/_index.html.erb where line #16 raised:
> `<%= link_to field.attribute, attachment_url(field.parent_instance, field.attribute) %>`

So you're referencing `field.parent_instance` on line 16 of `app/views/fields/refile/_index.html.erb`. Where does the `parent_instance` method come from? I can't find any reference to it anywhere on the internet. 

For the record the form and show views display just fine, the problem only occurs when I visit the index for any administrable resource (Posts in my case).

**Comment by acrogenesis** [comment]
You are right, the method `parent_instance` doesn't make any sense. The gem actually isn't prepared for handling the index page. If you have an idea on how to implement this will accept the PR 👍
